crates/propolis-api-types/src/lib.rs

```diff
@@ -159,12 +159,12 @@ pub struct InstanceSpecGetResponse {
 
 #[derive(Clone, Deserialize, Serialize, JsonSchema)]
 pub struct InstanceStateMonitorRequest {
-    pub gen: u64,
+    pub r#gen: u64,
```

Comment:
Is this a 2024-edition-preparation thing?

Reply:
Yep. I'll do this in another PR.